geth/utils/validation.py

```diff
@@ -71,6 +72,14 @@ def validate_geth_kwargs(geth_kwargs: GethKwargsTypedDict) -> None:
         raise PyGethValueError(f"error while validating geth_kwargs: {e}")
 
 
+class DefaultBlobSchedule(BaseModel):
```

Comment:
Is this necessary? If we set the default values in the `genesis.json` and only validate that against the model here, do we need to re-construct the values here?

Reply:
Having it be a separate model is not necessary. It could just as easily be:

```python
class GenesisDataConfig(BaseModel):
    chainId: int = 0
    ...
    cancunTime: int = 0
    # blobs
    blobSchedule: dict[str, Any] = {
        cancun: dict[str, Any] = {
            "target": 3,
            "max": 6,
            "baseFeeUpdateFraction": 3338477,
        },
        prague: dict[str, Any] = {
            "target": 6,
            "max": 9,
            "baseFeeUpdateFraction": whatever,
    }
```

It's just how I did it.

The default values here are just that, values that get filled if they are not present in the genesis.json provided by the user. What those values are, i.e. what constitutes a minimum viable `GenesisData`, should probably be clarified.

Reply:
If we are, as you mention in #255, overriding fields provided in `genesis.json` with default data from `GenesisData` or `GenesisDataConfig`, then yes, that is a problem.

Reply:
It's more about the fact that we already get these values from the `genesis.json`. So we are kind of doing double the "defaulting" by also setting them here in this class. I think validation of the `genesis.json` values is all that needs to be done here ideally. Am I missing something?